src/test/java/core/basesyntax/service/impl/ParserServiceImplTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ParserService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ParserServiceImplTest {
+    private static final String FRUIT_NAME = "banana";
+    private static final int FRUIT_QUANTITY = 80;
```

Comment:
you use it only one time in your code, try to avoid creating static fields in your tests, besides your testing service